### PR TITLE
format currencies in fields

### PR DESCRIPTION
**Pull request by axemonkey**

### CODE REVIEW

**Comment by tyom** [comment]
retest this please

File: cla_public/static-src/javascripts/modules/currency-formatting.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+(function () {
+  'use strict';
+
+  moj.Modules.CurrencyFormatting = {
+    currencyFields: [],
+
+    init: function() {
+      var self = this;
```
Comment (tyom) [inline cla_public/static-src/javascripts/modules/currency-formatting.js]: You're not using variable anywhere.

File: cla_public/static-src/javascripts/modules/currency-formatting.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    },
+
+    formatCurrency: function(evt) {
+      var $field = $(evt.target);
+      var pounds = $field.val().toString().replace(/,/g, '');
+
+      if(!isNaN(parseFloat(pounds))) {
+        var pence = Math.round(parseFloat(pounds)*100);
+        var pennies = pence % 100;
+        var pounds = Math.floor(pence/100);
```
Comment (tyom) [inline cla_public/static-src/javascripts/modules/currency-formatting.js]: `pounds` variable is already declared above.

Comment (axemonkey) [inline cla_public/static-src/javascripts/modules/currency-formatting.js]: DOH. Friday code. Will fix.

**Comment by tyom** [comment]
Awesome!

**Pull request merged**